helm/event-exporter-app/Chart.yaml

```diff
@@ -1,10 +1,10 @@
-apiVersion: v1
+apiVersion: v2
 name: event-exporter-app
-version: [[ .Version ]]
-appVersion: v0.10
-description: The event-exporter-app runs on Control Planes to watch and act upon Kubernetes events.
-home: https://github.com/opsgenie/kubernetes-event-exporter
+description: The Event Exporter runs on management clusters to watch and act upon Kubernetes events.
+version: 0.14.0
+appVersion: "0.9"
```

Comment:
Yes, because it was wrongly set in the `Chart.yaml` but never bumped in the `values.yaml`, which is being used in the Deployment. I still need to bump it in the `values.yaml` and `Chart.yaml`, but in a different PR.